vignettes/web_only/BCRpipeline.Rmd

```diff
@@ -233,14 +239,81 @@ bcr$full_clones$Cluster %>% unique()
 We have found mismatches between a germline and an ancestor sequence. Dots represent nucleotides matches between the sequences, letters represent mismatches between the sequences:
 
 ```{r example 12}
+# the example of common ancestor sequence
+bcr$full_clones$Common.Ancestor[1]
+
+# the example of mismatches between a germline and an ancestor sequence
 bcr$full_clones$Germline.Output[1]
 ```
 
 We have calculated a trunk length for each cluster:
 
 ```{r example 13}
-bcr$full_clones$Trunk.Length
+bcr$full_clones[ , c('Cluster', 'Trunk.Length') ]
 ```
+
+# Somatic hypermutation analysis
+
+Somatic hypermutation analysis is extremely useful for BCRs. The somatic hypermutation rate allows us to estimate the maturation of repertoire and figure out what kind of mutation contributes to the emergence of high-affinity antibodies.
```

Comment:
1. omit "extremely" or clarify
2. The rate of somatic hypermutation allows us to estimate repertoire maturation and detect the type of mutation contributing to the emergence on high affinity antibodies.

Reply:
The rate of somatic hypermutation allows us to estimate repertoire maturation and detect the type of mutation contributing to the emergence on high affinity antibodies. This makes somatic hypermutation analysis a valuable asset for B-cell repertoire analysis.

Reply:
The rate of somatic hypermutation allows us to estimate repertoire maturation and detect the type of mutation contributing to the emergence of high affinity antibodies.